source/Sound.Flac/Decoder.cs

```diff
@@ -179,7 +179,7 @@ internal static class Decoder {
 					 * block may be shorter than the stream blocksize; its starting sample number will be calculated as the frame number 
 					 * times the previous frame's blocksize, or zero if it is the first frame). 
 					 */
-					ulong sampleOrFrameNumber = (ulong)reader.ReadUTF8EncodedInteger();
+					reader.ReadUTF8EncodedInteger(); // ulong sampleOrFrameNumber = (ulong)
```

Comment:
We don't support variable frame lengths in the flack decoder plugin.

I suspect it'll fall over nastily if these are used, but haven't ever got as far as trying to debug it properly. (Know nothing about FLACs...)